components/ActivityForm/index.js

```diff
@@ -4,6 +4,15 @@ import categories from "@/assets/categories";
 import { useState } from "react";
 import Link from "next/link";
 import { StyledButton } from "../ActivityDetails";
+import countries from "@/assets/countries";
+import Select from "react-select";
```

Comment:
Hi, just out of curiosity: Why did you use the Select Component instead of self-written select element? What's the advantage?